### ISSUE TITLE
Cannot open remote Jupyter notebooks with Markdown

### ISSUE THREAD

**Issue by DoWalther** [microsoft/vscode-remote-release#5250]
## Environment data

-   VS Code version: 1.57.0 (Universal)
-   Jupyter Extension version (available under the Extensions sidebar): 2021.6.999
-   Python Extension version (available under the Extensions sidebar): 2021.6.0-rc
-   OS (Mac) and version: Darwin x64 20.3.0 - 20.5.0 (tested)
-   Python and/or Anaconda version: conda 4.10.0
-   Type of virtual environment used (virtualenv): Kubernetes Pod
-   Jupyter server running: Remote 

## Expected behaviour
Jupyter Notebooks should open normal and mardown cells should be shown.

## Actual behaviour
Creating new Notebooks works fine, when writing Markdown cells and execute them, the markdown cell disappears. When closing and reopening the Notebook, it loads infinte without any error message.

## Steps to reproduce:
1. Create a new notebook and write a markdown cell
2. Execute the markdown cell --> Markdown disappears
3. Save the notebook and reopen it.

![github_issue](https://user-images.githubusercontent.com/7305011/122539915-695e7780-d028-11eb-85ad-ea54cf28c760.gif)

This issue is resolved when disabeling the new extensible markdown renderer in the user settings.

**Comment by mjbvz** [comment]
@DoWalther Do you see this in [VS Code insiders](https://code.visualstudio.com/insiders/) (1.58)

**Comment by DoWalther** [comment]
@mjbvz Yes, same issue in VS Code insiders 1.58.0, all extensions up to date. 
Here is a screenshot when I execute the markdown cell and the developer tools consol output. 

![Screenshot 2021-07-01 at 19 26 56](https://user-images.githubusercontent.com/7305011/124165881-95570f80-daa2-11eb-9f52-91a23fbe7479.png)

```
notebookWebviewPreloads.js:3 Uncaught (in promise) TypeError: Failed to execute 'fetch' on 'Window': Failed to parse URL from https://vscode-remote%2Bk8s-container%2Bcontext%3Ddwalther%2Bpodname%3Ddwa-conda%2Bnamespace%3Dresearch%2Bname%3Ddwalther-tf%2Bimage%3Dfoo-bar.de%253a30500%252fconda-kite%253a1.01.vscode-resource.vscode-webview.net/home/dwalther/.vscode-server-insiders/bin/1fc68f3cf274141942b424894645a4f7096ec1f1-insider/extensions/markdown-language-features/notebook-out/index.js
    at b (notebookWebviewPreloads.js:3)
    at r (notebookWebviewPreloads.js:3)
    at x._load (notebookWebviewPreloads.js:3)
    at x.load (notebookWebviewPreloads.js:3)
    at notebookWebviewPreloads.js:3
    at Array.map (<anonymous>)
    at Object.render (notebookWebviewPreloads.js:3)
    at H.updateContentAndRender (notebookWebviewPreloads.js:3)
    at new H (notebookWebviewPreloads.js:3)
    at Object.createMarkupCell (notebookWebviewPreloads.js:3)
```

**Comment by DoWalther** [comment]
Seems like this issue is resolved with the latest VS Code Insiders 1.60.0. I can open Notebooks with Markdowns and they are displayed correctly.

**Comment by mjbvz** [comment]
Thanks for confirming. I believe this has the same root cause as Microsoft/vscode#123494 

Please let me know if this issue comes up again
